src/handlers/notification/index.js

```diff
@@ -635,17 +635,27 @@ const processMessage = async (msg, span) => {
 
     const { url: callbackURLTo } = await getEndpointFn(destination, REQUEST_TYPE.PATCH, true)
     const endpointTemplate = getEndpointTemplate(REQUEST_TYPE.PATCH)
-    headers = createCallbackHeaders({ headers: content.headers, httpMethod: PATCH, endpointTemplate })
-    logger.debug(`Notification::processMessage - Callback.sendRequest({ ${callbackURLTo}, ${PATCH}, ${JSON.stringify(content.headers)}, ${payload}, ${id}, ${source}, ${destination} ${hubNameRegex} })`)
+
+    let payloadForFXP = JSON.parse(payload)
+    if (payloadForFXP.fulfilment) {
+      delete payloadForFXP.fulfilment
```

Comment:
It's a bit unclear for me, why we need to delete _fulfilment_ from _payloadForFXP_

Reply:
HI @geka-evk as per the spec, there is no fulfilment in the patch notification.

Reply:
https://github.com/mojaloop/api-snippets/blob/7975487b3dc88a842f833bf4dd46ba861b4289fc/docs/fspiop-rest-v2.0-openapi3-snippets.yaml#L3421